.github/CONTRIBUTING.md

```diff
@@ -8,6 +8,14 @@ Remember, you do not have to be a Symfony guru or even a programmer to help!
 You can learn [how to contribute the patches](https://docs.sylius.com/en/latest/contributing/code/patches.html)
 in our [Contributing Guide](https://docs.sylius.com/en/latest/contributing/index.html).
 
+Translations
+------------
+
+If you find a typo in English, please follow the contribution guidelines as specified above.  
+If you find a typo or a missing translation in another language than English: [go to Crowdin](https://crowdin.com/project/sylius) and suggest your changes!
+
+We really appreciate your help, thank you so much for making Sylius international!
```

Comment:
Can we add a link to https://docs.sylius.com/en/latest/book/contributing/translations/index.html?